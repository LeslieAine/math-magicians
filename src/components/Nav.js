import { BrowserRouter, Link, Outlet } from 'react-router-dom';
import styles from '../styles/Nav.module.css';

const Nav = () => (
  <>
    <nav className={styles.nav}>
      <h1 className={styles.title}>Math Magicians</h1>
      <BrowserRouter>
        <ul className={styles.navList}>
          <li>
            <Link to="/">Home</Link>
          </li>
          <li>
            <Link to="/calculator">Calculator</Link>
          </li>
          <li>
            <Link to="/quote">Quote</Link>
          </li>
        </ul>
      </BrowserRouter>
    </nav>
    <Outlet />
  </>
);

export default Nav;
