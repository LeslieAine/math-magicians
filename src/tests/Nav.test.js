import renderer from 'react-test-renderer';
import Nav from '../components/Nav';

it('renders correctly', () => {
  const tree = renderer.create(<Nav />).toJSON();
  expect(tree).toMatchSnapshot();
});
