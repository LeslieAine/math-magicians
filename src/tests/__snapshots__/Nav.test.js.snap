// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="nav"
>
  <h1
    className="title"
  >
    Math Magicians
  </h1>
  <ul
    className="navList"
  >
    <li>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;
